Sources/Models/DetectorType.swift

```diff
@@ -45,6 +45,32 @@ public enum DetectorType {
         case .phoneNumber: return .phoneNumber
         case .url: return .link
         case .transitInformation: return .transitInformation
+        case .custom: return .regularExpression
+        }
+    }
+
+    /// Simply check if the detector type is a .custom
+    public var isCustom: Bool {
+        switch self {
+        case .custom: return true
+        default: return false
+        }
+    }
+
+    ///The hashValue of the `DetectorType` so we can conform to `Hashable` and be sorted.
+    public var hashValue: Int {
+        return self.toInt()
+    }
+
+    /// Return an 'Int' value for each `DetectorType` type so `DetectorType` can conform to `Hashable`
+    private func toInt() -> Int {
```

Comment:
Are `toInt()` and `hashValue` being used somewhere?  I reviewed fast. Maybe I'm missing something

Reply:
It's use because since we have an enum with a type and we store theses `DectectorType` into a dictionary we need to implement the `Hashable` protocol